pkg/email/templates/login_token.html.tmpl

```diff
@@ -0,0 +1,25 @@
+{{- define "title" -}}
+Login Token
+{{- end -}}
+
+{{- define "preview" -}}
+A login token was requested for your user "{{ .Receiver.Ids.IDString }}".
+{{- end -}}
+
+{{- define "body" -}}
+<p>
+Dear {{ .ReceiverName }},
+</p>
+<p>
+A login token was requested for your user "{{ .Receiver.Ids.IDString }}" on {{ .Network.Name }}.
+</p>
+<p>
+You can now visit <a href="{{ .Network.IdentityServerURL }}/token-login?token={{ .LoginToken }}">this link</a> to log in.
```

Comment:
```suggestion
You can now visit <a href="{{ .Network.IdentityServerURL }}/token-login?token={{ .LoginToken }}">this link</a> to log in.<br/><br/>
```

Reply:
Not going to do this. Feel free to rewrite the HTML in a future PR if you feel like it.